config/production.yaml

```diff
@@ -0,0 +1,2 @@
+application:
```

Comment:
I think we're going to need to copy the config files (or full directory) inside of the `Dockerfile`, or else the files won't be there, or include them directly in the binary.